### PR TITLE
CA-527 image size reduce

### PR DESCRIPTION
**Pull request by dominicusplatus**

### CODE REVIEW

File: CollAction/Services/Image/AmazonS3ImageService.cs [hunk 1]
```diff
@@ -104,7 +107,12 @@ public async Task<Image<Rgba32>> UploadToImage(IFormFile fileUploaded)
                 using (MemoryStream ms = new MemoryStream())
                 {
                     await uploadStream.CopyToAsync(ms);
-                    return SixLabors.ImageSharp.Image.Load(ms.ToArray());
+                    var image = SixLabors.ImageSharp.Image.Load(ms.ToArray());
+                    var scaleRatio = GetScaleRatioForImage(image);
+                    image.Mutate(x => x
```
Comment (timstokman) [inline CollAction/Services/Image/AmazonS3ImageService.cs]: Only mutate when scale factor !+ 1.0.

File: CollAction/Services/Image/AmazonS3ImageService.cs [hunk 2]
```diff
@@ -118,6 +126,19 @@ private byte[] ConvertImageToPng(Image<Rgba32> image)
             }
         }
 
+        private double GetScaleRatioForImage(Image<Rgba32> image)
+        {
+            if(image.Width > _imageResizeThreshold || image.Height > _imageResizeThreshold){
```
Comment (timstokman) [inline CollAction/Services/Image/AmazonS3ImageService.cs]: Formatting, use [allmann braces](https://en.wikipedia.org/wiki/Indentation_style#Allman_style)

Comment (timstokman) [reply CollAction/Services/Image/AmazonS3ImageService.cs]: And spaces between ( and the "if"

File: CollAction/Services/Image/AmazonS3ImageService.cs [hunk 3]
```diff
@@ -118,6 +129,22 @@ private byte[] ConvertImageToPng(Image<Rgba32> image)
             }
         }
 
+        private double GetScaleRatioForImage(Image<Rgba32> image)
+        {
+            if (image.Width > _imageResizeThreshold || image.Height > _imageResizeThreshold){
```
Comment (timstokman) [inline CollAction/Services/Image/AmazonS3ImageService.cs]: The above line still has wrong braces formatting (must be on a newline)

File: CollAction/Services/Image/AmazonS3ImageService.cs (hunk 1 above)
Comment (dominicusplatus) [reply CollAction/Services/Image/AmazonS3ImageService.cs]: sory, fixed in 7a1eade

File: CollAction/Services/Image/AmazonS3ImageService.cs (hunk 3 above)
Comment (timstokman) [reply CollAction/Services/Image/AmazonS3ImageService.cs]: Maybe we start adding stylecop, then the compiler will start warning for badly formatted lines..

File: CollAction/Services/Image/AmazonS3ImageService.cs (hunk 2 above)
Comment (dominicusplatus) [reply CollAction/Services/Image/AmazonS3ImageService.cs]: sory, fixed in 7a1eade

File: CollAction/Services/Image/AmazonS3ImageService.cs (hunk 3 above)
Comment (dominicusplatus) [reply CollAction/Services/Image/AmazonS3ImageService.cs]: Sory, fixed in 7a1eade, will use Rider in future.

**Pull request merged**